build/go-runner/go.mod

```diff
@@ -0,0 +1,5 @@
+module k8s.io/kubernetes/build/go-runner
+
+go 1.14
```

Comment:
nit: go 1.14 but the builder is 1.13
IIRC this can cause misleading error messages.

Reply:
Fixed.